README.md

```diff
@@ -100,7 +100,10 @@ The current tools are the following:
 ## Developer Guide
 Please refer to the [Developer Guide](doc/developer.md) for details.
 
+## Slides
+[Presentation Slides](doc/TiefVision.pdf) with detailed view of inner workings of TiefVision, from a talk given by Pau Carre.
```

Comment:
nice, but would it be possible to add the google docs link so that we don't need to include the pdf binary?